ui/page/components/utils.go

```diff
@@ -112,22 +112,32 @@ func checksumByte(data []byte) byte {
 }
 
 func LayoutIconAndText(l *load.Load, gtx C, title string, val string, col color.NRGBA) D {
+	return layoutIconAndText(l, gtx, title, val, col, values.TextSize14, values.MarginPadding8)
+}
+
+func LayoutIconAndTextWithSize(l *load.Load, gtx C, title string, val string, col color.NRGBA, size unit.Sp, iconSize unit.Dp) D {
+	return layoutIconAndText(l, gtx, title, val, col, size, iconSize)
+}
+
+func layoutIconAndText(l *load.Load, gtx C, title string, val string, col color.NRGBA, size unit.Sp, iconSize unit.Dp) D {
```

Comment:
How about an `IconAndText` struct with properties for `title string, val string, col color.NRGBA, size unit.Sp, iconSize unit.Dp`. When an instance of the struct is initialized, set the size to values.TextSize14 and the iconSize to values.MarginPadding8. Callers who want to use a different value can change those default values after initializing the object. Then you can have a single `IconAndText.Layout(gtx C)` method.

Reply:
By the way, there should also be an `icon` property. The default value can be `cryptomaterial.NewIcon(l.Theme.Icons.ImageBrightness1)`.

Reply:
@itswisdomagain It's fine for now. If necessary we can build an IconAndText component later for a more general case. If we only build it for what is needed now, I don't think it's necessary.

Reply:
I wasn’t recommending a struct because of a general use case though; it’s to make the code cleaner. It’s definitely better than these several functions we have, don’t you think?